squap/.htaccess

```diff
@@ -0,0 +1,4 @@
+Header set Access-Control-Allow-Origin *
+Options +FollowSymLinks
+RewriteEngine on
+RewriteRule ^/?$ http://stlab.istc.cnr.it/squap [R=302,L]
```

Comment:
Did you mean to also have a rule that would redir to the owl file at that url?

Reply:
Yes, depending on the accept header of the request I'd like to serve HTML or one OWL serialisation.

Reply:
Do you plan to do the redirection on w3id, or do you plan to do it on your server? If the former, you will need to add a 303 redirection (there are plenty of examples in w3id repo, e.g., https://github.com/perma-id/w3id.org/blob/master/games/spec/.htaccess) instead of the 302 redirection

Reply:
I plan to do the redirection on my server.

Reply:
Ok, then we can merge. Please just consider the https comment @davidlehn was doing regarding your readme

Reply:
Sure.